Src/StdLib/Lib/ensurepip/__init__.py

```diff
@@ -104,6 +104,8 @@ def bootstrap(*, root=None, upgrade=False, user=False,
 
         # Construct the arguments to be passed to the pip command
         args = ["install", "--no-index", "--find-links", tmpdir]
+        if sys.platform == "cli":
```

Comment:
Can we use `sys.implementation.name == "ironpython"` for this instead of `sys.platform`?